contracts/smart-contract-wallet/aa-4337/test/TestSignatureAggregator.sol

```diff
@@ -4,26 +4,28 @@ pragma solidity 0.8.17;
 /* solhint-disable reason-string */
 
 import "../interfaces/IAggregator.sol";
-import "hardhat/console.sol";
-import "./SimpleAccount.sol";
+import "../interfaces/IEntryPoint.sol";
+import "../samples/SimpleAccount.sol";
 
 /**
  * test signature aggregator.
  * the aggregated signature is the SUM of the nonce fields..
  */
 contract TestSignatureAggregator is IAggregator {
 
+    /// @inheritdoc IAggregator
     function validateSignatures(UserOperation[] calldata userOps, bytes calldata signature) external pure override {
-        uint256 sum = 0;
-        for (uint256 i = 0; i < userOps.length; i++) {
-            uint256 nonce = userOps[i].nonce;
+        uint sum = 0;
+        for (uint i = 0; i < userOps.length; i++) {
+            uint nonce = userOps[i].nonce;
             sum += nonce;
         }
-        require(signature.length == 32, "TestSignatureValidator: sig must be uint256");
-        (uint256 sig) = abi.decode(signature, (uint256));
+        require(signature.length == 32, "TestSignatureValidator: sig must be uint");
+        (uint sig) = abi.decode(signature, (uint));
```

Comment:
same for these changes, had decided to use the uint256

Reply:
its just test contract from infinitism samples so imo shouldn't matter